the-collab-lab/tcl-78-smart-shopping-list
# reinstall tailwind

## Description

Tailwind has been reinstalled to ensure proper configuration.

## Related Issue

closes #47 

## Acceptance Criteria

- [x] Tailwind CSS has been uninstalled and reinstalled successfully
- [x] Applied Tailwind styles are working as expected

## Type of Changes

`bug fix`

## Testing Steps / QA Criteria

1. pull the newest branch
2. `npm install`
3. apply any styling to a specific element (e.g. for a paragraph use `<p className="text-5xl text-red-500 underline">`)
4. `npm start`
5. navigate to browser to confirm the new styling is applied
